lib/Dancer2/Core/Role/HasLocation.pm

```diff
@@ -33,7 +33,10 @@ sub _build_location {
     my $script = $self->caller;
 
     # default to the dir that contains the script...
-    my $location = Dancer2::FileUtils::dirname($script);
+    my $location = Path::Tiny::path($script)->parent;
+
+    $location->is_dir
+        or Carp::croak("Caller $script is not an existing file");
```

Comment:
@xsawyerx this is causing the only current (non-Windows) test failure, and was the only failure before I rebased. I know it is a long time ago, but can you remember why this was added?

Reply:
Hmm..... no memory at all.

I want to get back to this, but I can't promise when. Maybe Saturday?

Reply:
@xsawyerx Saturday would be cool, but no sweat. With luck I'll have time to look at the Windows test failures this weekend.